### PR TITLE
feat(codewhisperer): roll out classifier to more languages

### PR DESCRIPTION
**Pull request by zixlin7**
## Problem

- Roll out Classifier trigger 100% for languages
- Fix telemetry for classifier result inaccurate due to blocked triggers

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Will-ShaoHua — APPROVED** [review]

File: src/codewhisperer/service/inlineCompletionService.ts [hunk 1]
```diff
@@ -392,11 +393,19 @@ export class InlineCompletionService {
...
     ) {
         if (vsCodeState.isCodeWhispererEditing || this._isPaginationRunning || this.isSuggestionVisible()) {
             return
         }
+        if (ClassifierTrigger.instance.shouldInvokeClassifier(editor.document.languageId)) {
+            ClassifierTrigger.instance.recordClassifierResultForAutoTrigger(editor, autoTriggerType, event)
+        }
+        const triggerChar = event?.contentChanges[0]?.text
+        if (autoTriggerType === 'SpecialCharacters' && triggerChar) {
+            TelemetryHelper.instance.setTriggerCharForUserTriggerDecision(triggerChar)
```
Comment (Will-ShaoHua) [inline src/codewhisperer/service/inlineCompletionService.ts:407]: I thought we already sent special char before this change

Comment (Will-ShaoHua) [reply src/codewhisperer/service/inlineCompletionService.ts:407]: didn't we?

Comment (zixlin7) [reply src/codewhisperer/service/inlineCompletionService.ts:407]: we did but it probably suffer from the same bug so I moved it here as well

**Review by justinmk3 — APPROVED** [review]

**Comment by justinmk3** [comment]
Failing test:

    src/test/codewhisperer/service/keyStrokeHandler.test.ts

**Pull request merged**